### ISSUE TITLE
Vscode extension does not install dafny correctly

### ISSUE THREAD

**Issue by emmet-m** [DafnyVSCode/Dafny-VSCode#49]
Hi All,

I just installed the vscode extension on Ubuntu 16.04, when prompted if I wanted to install Dafny I got the following error message:

```
Could not get Dafny Release assets from JSON response.
errrroooorrr: Could not get Dafny Release assets from JSON response.
[Error - 4:14:19 pm] Request textDocument/codeLens failed.
  Message: Request textDocument/codeLens failed unexpectedly without providing any details.
  Code: -32603
```

Furthermore, when I enter in a DafnyServer.exe path incorrectly, the extension asks me if I would like to install dafny (as opposed to showing an error that the path is invalid/doesn't contain the server).

I managed to install it by downloading the release zip and entering the path myself, but I was just wondering if the extension was broken or if it didn't work on my system?

Thanks!

**Comment by emmet-m** [comment]
Just a follow up, I've tried this again on Ubuntu18.04 on my desktop computer and I get the same error

**Comment by fabianhauser** [comment]
Hi @emmet-m 

Thanks for reporting. This error is due to the very recent move of the Dafny-Repository into its own organisation (now https://github.com/dafny-lang/dafny), which must be corrected in the plugin.

**Comment by fabianhauser** [comment]
I just released version 0.17.1, which includes the new repository url.

@emmet-m: could you please check if the new release fixes the issue?

**Comment by emmet-m** [comment]
Hi Fabian, that indeed works! Thank you very much :)
